tools/bazel-lldb-wrapper.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# A wrapper script so workerd can be debugged with lldb, if invoked via bazel run.
+#
+# Example command-line invocation:
+#
+# bazel run -c dbg --spawn_strategy=local --features=oso_prefix_is_pwd --run_under \
```

Comment:
[This comment](https://github.com/bazelbuild/bazel/issues/6327#issuecomment-982965782) suggests to me that the `features` flag alone is sufficient and we should be able to at least do away with the `local` `spawn_strategy`?

Reply:
Interesting, I cannot tell you how much time I wasted with this issue. Bazel...it does indeed appear redundant.